website/content/api-docs/txn.mdx

```diff
@@ -266,20 +266,21 @@ look like this:
 The following tables summarize the available verbs and the fields that apply to
 those operations ("X" means a field is required and "O" means it is optional):
 
-| Verb               | Operation                               | Key | Value | Flags | Index | Session |
-| ------------------ | --------------------------------------- | :-: | :---: | :---: | :---: | :-----: |
-| `set`              | Sets the `Key` to the given `Value`     | `x` |  `x`  |  `o`  |       |         |
-| `cas`              | Sets, but with CAS semantics            | `x` |  `x`  |  `o`  |  `x`  |         |
-| `lock`             | Lock with the given `Session`           | `x` |  `x`  |  `o`  |       |   `x`   |
-| `unlock`           | Unlock with the given `Session`         | `x` |  `x`  |  `o`  |       |   `x`   |
-| `get`              | Get the key, fails if it does not exist | `x` |       |       |       |         |
-| `get-tree`         | Gets all keys with the prefix           | `x` |       |       |       |         |
-| `check-index`      | Fail if modify index != index           | `x` |       |       |  `x`  |         |
-| `check-session`    | Fail if not locked by session           | `x` |       |       |       |   `x`   |
-| `check-not-exists` | Fail if key exists                      | `x` |       |       |       |         |
-| `delete`           | Delete the key                          | `x` |       |       |       |         |
-| `delete-tree`      | Delete all keys with a prefix           | `x` |       |       |       |         |
-| `delete-cas`       | Delete, but with CAS semantics          | `x` |       |       |  `x`  |         |
+| Verb               | Operation                                | Key | Value | Flags | Index | Session |
+| ------------------ | ---------------------------------------- | :-: | :---: | :---: | :---: | :-----: |
+| `set`              | Sets the `Key` to the given `Value`      | `x` |  `x`  |  `o`  |       |         |
+| `cas`              | Sets, but with CAS semantics             | `x` |  `x`  |  `o`  |  `x`  |         |
+| `lock`             | Lock with the given `Session`            | `x` |  `x`  |  `o`  |       |   `x`   |
+| `unlock`           | Unlock with the given `Session`          | `x` |  `x`  |  `o`  |       |   `x`   |
+| `get`              | Get the key, fails if it does not exist  | `x` |       |       |       |         |
+| `get-not-exists`   | Get the key, or nil if it does not exist | `x` |       |       |       |         |
```

Comment:
nit: the example in the API docs are referring to JSON response values where we mention in the docs above this that the `Value` field is "a base64 encoded string or null". Should we be consistent and say `null` rather than `nil` here?